core/quality/mysterium_morqa.go

```diff
@@ -174,7 +164,9 @@ func (m *MysteriumMORQA) sendMetrics() error {
 		return err
 	}
 
-	defer response.Body.Close()
+	defer func() {
+		_ = response.Body.Close()
```

Comment:
Can't we adjust the linter and at least make it ignore defers like these? This is so verbose

Reply:
Turns out no need to adjust linter, i just misunderstand linter warning